backend/app/api/v1/lantern_api.py

```
@@ -36,20 +37,29 @@ async def music_status(
-    )
+    ),
+    resume: bool = Query(False, description="이전 이벤트 ID를 기반으로 이어받을지 여부"),
+    last_event_id: Optional[str] = Header(None, convert_underscores=False)
 ):
     db = get_mongo_client(request)
     repo = LanternRepository(db)
     start_time = time.time()
     sent_task_ids = set()
+
+    if resume and last_event_id:
+        valid_ids = {s["task_id"] for s in doc.get("music_statuses", [])}
```

Ruff (0.11.9)

Undefined name `doc`

(F821)